devtools/common/src/main/filtered/quarkus.properties

```diff
@@ -29,3 +29,6 @@ plugin-artifactId=quarkus-maven-plugin
 plugin-version=${project.version}
 
 supported-maven-versions=${supported-maven-versions}
+# the proposed version must be in the range of the supported versions
+proposed-maven-version=3.5.4
+maven-wrapper-version=0.7.4
```

Comment:
Ah ah, you went half way. I think these properties should be in our pom (see where supported-maven-versions is). BTW, why not use 3.6.0?

Reply:
I intentionally didn't add them in the pom since I didn't feel they belonged there. If you think it's best, I will of course add them

Reply:
I suggested `3.5.4` since that is what I definitely know is working. If we are certain that everything works with `3.6.0` (which in all likelihood it is), then I can set that.
Furthermore I wasn't able to understand what version our CI uses.

Reply:
I see from the CI logs that Maven version is `3.6.0`, so I will use it.
WDYT about my comment above about the properties @gsmet ?

Reply:
After debating it more with myself I see the light :). Properties added to pom